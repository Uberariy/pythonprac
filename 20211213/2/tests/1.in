[22, 58, 95, 33, 47, 11, 76, 38, 70, 84, 40, 35, 56, 28, 13, 23, 1, 3, 5, 2, 4]