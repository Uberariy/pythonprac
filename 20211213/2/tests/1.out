[1, 2, 3, 4, 5, 11, 13, 22, 23, 28, 33, 35, 38, 40, 47, 56, 58, 70, 76, 84, 95]
